### PR TITLE
[Docs]: turbot_ldap_directory and turbot_group_profile

### PR DESCRIPTION
**Pull request by RupeshPatil20**

### CODE REVIEW

File: website/docs/r/group_profile.html.markdown [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+layout: "turbot"
+title: turbot
+template: Documentation
+page_title: "Turbot: turbot__group_profile"
+nav:
+  title: turbot__group_profile
```
Comment (e-gineer) [inline website/docs/r/group_profile.html.markdown]: This title is wrong?

File: website/docs/r/group_profile.html.markdown [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## Example Usage
+
+**Creating Your First group profile**
+
+```hcl
+resource "turbot_group_profile" "test" {
+  title               = "terraform test group profile"
+  directory           = "112233445566"
+  group_profile_id    = "test"
```
Comment (e-gineer) [inline website/docs/r/group_profile.html.markdown]: too much whitespace before = signs - should be min while aligned

File: website/docs/r/group_profile.html.markdown [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+title: turbot
+template: Documentation
+page_title: "Turbot: turbot__group_profile"
+nav:
+  title: turbot__group_profile
+---
+
+# turbot_profile
+
+The `Turbot group profile` resource adds support for creating group profile under a directory type. It is used to create, manage and delete group profiles.
```
Comment (e-gineer) [inline website/docs/r/group_profile.html.markdown:12]: I don't understand when I would use a group profile / create one in terraform. Please confirm with Tuhin or Mike for why / the use case.

File: website/docs/r/group_profile.html.markdown [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+- `directory` - (Required) The parent directory of the group profile, either as an id, or an AKA.
+- `group_profile_id` - (Required)  The unique identifier of the group profile. For new group profiles this must be unique for the parent directory.
+- `title` - (Required)  The title of the group profile.
+
+## Attributes Reference
+
+In addition to all the arguments above, the following attributes are exported:
+
+- `id` - Unique identifier of the resource.
+- `status` - Status of the group profile. Probable options are `ACTIVE`, `INACTIVE` and.
```
Comment (e-gineer) [inline website/docs/r/group_profile.html.markdown]: unfinished sentance

"probable" is a weird word in this context, please look for something better - perhaps "Available" or "Valid"

File: website/docs/r/group_profile.html.markdown (hunk 1 above)
Comment (e-gineer) [reply website/docs/r/group_profile.html.markdown]: These terraform internal strings like turbot__group_profile are not helpful for users in any scenario. Suggest we stop using them in any docs / release notes (unless it's really a TF standard doc format).

File: website/docs/r/group_profile.html.markdown [hunk 5]
```diff
@@ -0,0 +1,47 @@
+---
+layout: "turbot"
+title: turbot
+template: Documentation
+page_title: "Turbot: turbot__group_profile"
```
Comment (e-gineer) [inline website/docs/r/group_profile.html.markdown]: What is the page_title? I don't think this is valid

File: website/docs/r/group_profile.html.markdown [hunk 6]
```diff
@@ -0,0 +1,47 @@
+---
+layout: "turbot"
+title: turbot
```
Comment (e-gineer) [inline website/docs/r/group_profile.html.markdown:3]: This title is wrong

File: website/docs/r/group_profile.html.markdown [hunk 7]
```diff
@@ -0,0 +1,47 @@
+---
+layout: "turbot"
```
Comment (e-gineer) [inline website/docs/r/group_profile.html.markdown:2]: What is the layout field?

File: website/docs/r/group_profile.html.markdown [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+title: turbot
+template: Documentation
+page_title: "Turbot: turbot__group_profile"
+nav:
+  title: turbot__group_profile
+---
+
+# turbot_profile
```
Comment (e-gineer) [inline website/docs/r/group_profile.html.markdown]: This heading is wrong

File: website/docs/r/group_profile.html.markdown [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+
+The `Turbot group profile` resource adds support for creating group profile under a directory type. It is used to create, manage and delete group profiles.
+
+## Example Usage
+
+**Creating Your First group profile**
```
Comment (e-gineer) [inline website/docs/r/group_profile.html.markdown]: why is this half capitlized?

File: website/docs/r/group_profile.html.markdown [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+
+```hcl
+resource "turbot_group_profile" "test" {
```
Comment (e-gineer) [inline website/docs/r/group_profile.html.markdown]: Please try to use examples that have more real life meaning. A good example / good name can be half the documentation without me reading any descriptions etc below.

when it's just "foo" or "test" or generic, it doesn't really teach me anything in reading the example.

e.g. compare to if this doc referred to an "Admins" group?

File: website/docs/r/group_profile.html.markdown [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+  title               = "terraform test group profile"
+  directory           = "112233445566"
```
Comment (e-gineer) [inline website/docs/r/group_profile.html.markdown]: AKAs are better than IDs in most documentation, please use a sample AKA here instead.

File: website/docs/r/group_profile.html.markdown [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+  group_profile_id    = "test"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- `directory` - (Required) The parent directory of the group profile, either as an id, or an AKA.
+- `group_profile_id` - (Required)  The unique identifier of the group profile. For new group profiles this must be unique for the parent directory.
```
Comment (e-gineer) [inline website/docs/r/group_profile.html.markdown]: Is it really called `group_profile_id`? It's not actually an ID :-( ... It's a name.

Might be nothing we can do now, but this is not consistent with anything else.....

File: website/docs/r/group_profile.html.markdown (hunk 7 above)
Comment (RupeshPatil20) [reply website/docs/r/group_profile.html.markdown:2]: layout  - Which navigation sidebar to display for this page. A layout called <NAME> gets loaded from `/content/source/layouts/<NAME>.erb`

In our case NAME is turbot

**Comment by RupeshPatil20** [comment]
Group Profile resource is no more a requirement, Ldap was close under #126

**Pull request closed without merging**